app/models/discovery_rule.rb

```diff
@@ -36,13 +36,31 @@ class DiscoveryRule < ApplicationRecord
     end
   }
 
+  def unique_priority
+    Taxonomy.enabled_taxonomies.each do |enabled_taxonomy|
```

Comment:
@rahulbajaj0509,
Here, it iterates on `Taxonomy.enabled_taxonomies`. Let's say  `organizations, locations` both are enabled.
But then why the method [suggest_priority](https://github.com/theforeman/foreman_discovery/pull/450/files#diff-dce2846f7e5c2e776201ba13b1445290R61) considered only `organization`.

@lzap, please confirm that this is an expected behavior?

Reply:
You are right, only organizations are relevant in this case.